src/components/Clusters/ClusterConfigCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { Badge, Button, Card, Text } from '@mantine/core';
+import { format } from 'date-fns';
+import { RiSettings4Line } from 'react-icons/ri';
+
+import { useAuth } from '@/hooks/useAuth';
+import type { Cluster } from '@/types/cluster';
+
+function ClusterConfigCard({ cluster }: { cluster: Cluster }) {
+  const { user } = useAuth();
+  const jar = cluster?.jars[cluster.jars.length - 1];
+
+  const stageValues = Object.values(jar.schedulingInfo.stages);
+  const formattedDateAndTime = jar.uploadedAt ? format(jar.uploadedAt, 'MMM d yyyy, hh:ss a') : '';
+
+  const { strategy, configString } = cluster.migrationConfig;
+  function decodeConfigString() {
```

Comment:
nit: Prefer array functions inside a component

Reply:
Also not too sure what this function is meant to do but I can take a deeper look later.

Reply:
Basically this function is converting this type of string `"configString": {\"percentToMove\":25,\"intervalMs\":60000}"` into json object to be able to use percentToMove and intervalMs by replacing \" to ".

Reply:
If you want to turn it into an object why not just use JSON.parse()?

Reply:
> > nit: Prefer array functions inside a component
> 
> didn't understand this?
Basically,
const decodeConfigString = () => {}

Reply:
> If you want to turn it into an object why not just use JSON.parse()?

I am referring to https://netflix.github.io/mantis/reference/api/#get-information-about-a-particular-job. Here due to backslash in front of `"` this becomes escape sequence hence not able to parse it correctly. Although the mock fixtures in the codebase don't have these because of lint formatting(on save it automatically removes backslash).
<img width="435" alt="Screenshot 2023-04-27 at 7 55 35 PM" src="https://user-images.githubusercontent.com/92316166/234892847-2a4e09ec-f99d-410f-97b6-d91d043393f2.png">

For now, I am making changes and keeping it simple as JSON.parse() and if problem comes we can change it accordingly when I have access to true api requests instead of mirage one.